doc/syscalls/90001_screen.md

```diff
@@ -0,0 +1,220 @@
+---
+driver number: 0x90001
+---
+
+# Screen
+
+## Overview
+
+The screen driver allows the process to write data to a framebuffer of a screen.
+
+## Command
+
+  * ### Command number: `0`
+
+    **Description**: Does the driver exist?
+
+    **Argument 1**: unused
+
+    **Argument 2**: unused
+
+    **Returns**: SUCCESS if it exists, otherwise ENODEVICE
+
+  * ### Command number: `1`
+
+    **Description**: Checks if the Setup API is available
+
+    **Argument 1**: unused
+
+    **Argument 2**: unused
+
+    **Returns**: SUCCESS_U32 with 1 if yes, and 0 if no
+
+  * ### Command number: `3`
+
+    **Description**: Set brightness
+
+    **Argument 1**: Percent of brightness, 0% should turn off the screen, greater than 0% should turn it on.
```

Comment:
The might be the wrong place to flag this, but I don't think that brightness should turn on/off a screen. In well-lit conditions (waves from sunny San Diego..), brightness can be 0% (LCD backlight completely off) and the screen be perfectly usable.

The question of whether to power on/off the drawing of a screen should be its own dedicatd call.